tests/providers/amazon/aws/hooks/test_emr.py

```diff
@@ -191,6 +201,24 @@ def test_get_cluster_id_by_name(self):
 
         assert no_match is None
 
+        with pytest.raises(AirflowException):
```

Comment:
LGTM but I would move it to a separate test for clarity purposes. The shorter the tests are, the more likely people will read them, the better :)

Reply:
Addressed